src/modules/MarketHeroModule.js

```diff
@@ -0,0 +1,24 @@
+import { MODULE_MARKET_HERO } from '../analyticsTypes'
+import BasicModule from './BasicModule'
+
+export default class MarketHeroModule extends BasicModule {
+
+    constructor () {
+        super(MODULE_MARKET_HERO)
+    }
+
+    init (initConf = {}) {
+      // Load Market Hero
+      (function () {
+        var e = document.createElement("script");
+        e.src = (`https://176906.tracking.markethero.io/v1/lst/universal-script?ph=c9d23b7724050e2e2d267890a0e0da92a1dd4fc9c4ea7a4b4ea62fcc9f473d78&tag=!hyros%22;head.appendChild(script)`);
```

Comment:
Looks like this URL contains identifiers specific to our account. If that's the case, can you parametrise those? Would be best to avoid having our account information in public domain,

Reply:
yep done